pkg/controller/account/account_controller.go

```diff
@@ -171,14 +171,30 @@ func (r *ReconcileAccount) Reconcile(request reconcile.Request) (reconcile.Resul
 		return reconcile.Result{}, nil
 	}
 
-	// Ignore accounts for which we're (asynchronously) initializing regions -- unless they've been
-	// in that state for too long.
+	// Detect accounts for which we kicked off asynchronous region initialization
 	if accountIsInitializingRegions(currentAcctInstance) {
+		// Detect whether we set the InitializingRegions condition in *this* invocation of the
+		// operator or a previous one.
+		if regionInitStale(currentAcctInstance) {
+			// This means the region init goroutine(s) for this account were still running when an
+			// earlier invocation of the operator died. We want to recover those, so set them back
+			// to Creating, which should cause us to hit the region init code path again.
+			msg := "Recovering from stale region initialization."
+			utils.SetAccountStatus(currentAcctInstance, msg, v1alpha1.AccountCreating, AccountCreating)
```

Comment:
We should reset the timestamp on this. Otherwise if we've been creating+initializing for more than 25 minutes but less than 100, we'll time out on `accountCreatingTooLong` and fail the account.

Reply:
Yeah, turns out `accountCreatingTooLong` uses the CR's creation time, not the `Creating` Condition's `lastTransitionTime`.